### PR TITLE
fix some circular deps

### PR DESCRIPTION
**Pull request by malgorithms**
@zapu  - I'm trying to clear up circular dependencies. This simple change converts this:

image

to this:

image

I think @maxtaco would like you to try to fix the deeper issues (dh.js <--> eddsa.js) -- so let's chat about that.

### CODE REVIEW

File: src/bn.iced [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 {nbv,nbi,BigInteger,nbits} = require 'bn'
-{buffer_to_ui8a} = require './util'
+{buffer_to_ui8a} = require('pgp-utils').util
```
Comment (zapu) [inline src/bn.iced:3]: i don't think it's used in this file, or am I missing something?

**Comment by zapu** [comment]
this definitely gets rid of some of the warnings, the remaining one about `Pair` bothers me more because it probably means busted code. I need to investigate whether it's possible to actually get to the affected function (`@subkey_algo` can return `NaclEddsa` which is undefined because of circular import).

**Review by zapu — APPROVED** [review]

File: src/bn.iced (hunk 1 above)
Comment (malgorithms) [reply src/bn.iced:3]: oh you're right - I spent the time fixing the require to not be circular...but didn't realize it isn't needed. I'll push a fix.

**Review by zapu — APPROVED** [review]
Also tried in kbweb, LGTM!

**Pull request merged**